eth/wallet.go

```diff
@@ -21,3 +24,7 @@ func NewHdWallet(rng io.Reader) *hdwallet.Wallet {
 	}
 	return hdw
 }
+
+func NewRandomAddress(rng *rand.Rand) common.Address {
```

Comment:
This function is also already implemented in `tee/test/transaction.go`. Would be nice to remove it from there if possible.

Would need to touch the `txCache` test then IIRC.

Reply:
Ah forgot about that one, will remove my implementation then.